src/components/FilterHeaderCategories.js

```diff
@@ -0,0 +1,112 @@
+// @flow
+import React from "react";
+import { View, StyleSheet, TouchableHighlight, Image } from "react-native";
+import type { StyleObj } from "react-native/Libraries/StyleSheet/StyleSheetTypes";
+import {
+  interestButtonBgColor,
+  interestButtonTextColor,
+  filterShowMeTextColor,
+  categoriesFilterButtonBgColor
+} from "../constants/colors";
+import text from "../constants/text";
+import Text from "./Text";
+import CategoriesPills from "./CategoriesPills";
+import chevronRightImg from "../../assets/images/chevronRight.png";
+
+type Props = {
+  selectedCategories: Set<string>,
+  onFilterPress: Function
+};
+
+type CategoriesFilterButtonProps = {
+  style?: StyleObj,
+  onPress: Function
+};
+
+const CategoriesFilterButton = ({
+  style,
+  onPress
+}: CategoriesFilterButtonProps) => (
+  <TouchableHighlight
```

Comment:
Can you use our `Touchable` component here? It uses the native highlighting on Android, has the button accessibility trait by default and enforces a minimum size for the touch area.